grpc/core/src/test/java/io/helidon/grpc/core/GrpcTlsDescriptorTest.java

```diff
@@ -41,7 +41,7 @@ public void testDefaultSslDescriptor() {
 
     @Test
     public void testSslDescriptorWithBuilder() {
```

Comment:
Please have a look at `io.helidon.common.configurable.Resource` - that can be used to load a resource from various locations (classpath, file system, configuration value etc.) - also supports configuration.
This is used to locate such resources in other modules. Please replace your usage. 
We have an issue to improve the behavior of this class, we plan that for 2.0
This note belongs to `tlsCaCert`, `tlsCert` ans `tlsKey`